core/src/main/java/org/htsjdk/core/exception/HtsjdkException.java

```diff
@@ -0,0 +1,35 @@
+package org.htsjdk.core.exception;
+
+/**
+ * Exception type for all exceptions caused at runtime by HTSJDK-NEXT.

```

Comment:
We should decide on capitalization of these things somewhere.

Reply:
Agree - I vote for `HTSJDK-NEXT` for the project name in any documentation reference, and `Htsjdk` for prefix/suffix of classes requiring it. What do you thing?

Reply:
I definitely prefer `Htsjdk` in code, `HTSJDK-NEXT` seems very loud to me and implies that NEXT is some sort of acronym, but HTSJDK-Next seems a bit weird too.  Not sure.  I'll bow to popular vote.

Reply:
Maybe `HTSJDK-next`?

Reply:
I will change it to just say `HTSJDK`, as we might end up with htsjdk3 by discussions in Slack.